BangazonAPI/Controllers/ProductController.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using BangazonAPI.Models;
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+        public ProductController(IConfiguration config)
+        {
+            _config = config;
+        }
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+        [HttpGet]
+        public async Task<IActionResult> Get()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT Product.Id, Product.Price, 
+                                        Product.Title, Product.Description,
+                                        Product.Quantity, Product.Archived,
+                                        Product.CustomerId AS 'Customer Id',
+                                        Product.ProductTypeId AS 'Product Type Id'
+                                        FROM Product 
+                                        LEFT JOIN Customer on Product.CustomerId = Customer.Id
+                                        LEFT JOIN ProductType on Product.ProductTypeId = ProductType.Id";
+
+                    SqlDataReader reader = cmd.ExecuteReader();
```

Comment:
With soft delete-- products that are archived _shouldn't_ come back when you get the full list. It's okay if you support a singular `GET` for archived products, but they shouldn't appear in the `GET` all.